File: 3194-find-words-containing-character/find-words-containing-character.js
/**
 * @param {string[]} words
 * @param {character} x
 * @return {number[]}
 */
var findWordsContaining = function(words, x) {

  const indexed_array = new Array();

  for(let i = 0; i < words.length; i++) {

    if(words[i].includes(x)) {
        indexed_array.push(i);
    }
  }  

  return indexed_array;
};

File: 3194-find-words-containing-character/find-words-containing-character.cpp
class Solution {
public:
    vector<int> findWordsContaining(vector<string>& words, char x) {
        
        vector<int> indexed_array;

        for(int i = 0; i < words.size(); i++) {

            if(words[i].find(x) != string::npos)  indexed_array.push_back(i);
        }

        return indexed_array;
    }
};